conf/evolutions/default/85.sql

```diff
@@ -0,0 +1,12 @@
+# --- !Ups
+
+INSERT INTO `vitals` (`id`, `name`, `data_type`, `unit_of_measurement`)
+  VALUES ('11', 'weeksPregnant', 'float', 'weeks');
+
+# --- !Downs
+
+DELETE FROM `patient_encounter_vitals`
+WHERE `vital_id` = 11;
```

Comment:
After changing the above this should probably be:

DELETE FROM `patient_encounter_vitals` WHERE `vital_id` IN (SELECT id FROM vitals WHERE name = 'weeksPregnant')